src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const allMen = people.filter(person => person.sex === 'm');
+  let men;
+
+  century
+    ? men = allMen.filter(person => Math.ceil(person.died / 100) === century)
+    : men = allMen;
```

Comment:
```suggestion
  const men = century
    ? allMen.filter(person => Math.ceil(person.died / 100) === century)
    : allMen;
```
also you may do it inside 1 callback with previous filter